lib/avatar.cpp

```diff
@@ -152,7 +152,7 @@ bool Avatar::Private::checkUrl(const QUrl& url) const
 
     // FIXME: Make "mxc" a library-wide constant and maybe even make
     // the URL checker a Connection(?) method.
-    if (!url.isValid() || url.scheme() != "mxc" || url.path().count('/') != 1) {
+    if (!url.isValid() || url.scheme() != "mxc"_ls || url.path().count(QLatin1Char('/')) != 1) {
```

Comment:
Wouldn't
```suggestion
    if (!url.isValid() || url.scheme() != "mxc"_ls || url.path().count(u'/') != 1) {
```
work instead?